packages/core/echo/echo-db/src/echo-handler/echo-handler.test.ts

```diff
@@ -620,4 +620,15 @@ describe('Reactive Object with ECHO database', () => {
 
     expect(getDatabaseFromObject(another)).not.to.be.undefined;
   });
+
+  test('explicit reference API', async () => {
+    const { db, graph } = await builder.createDatabase();
+    graph.schemaRegistry.addSchema([TestSchemaType]);
+
+    const obj = db.add(create({ string: 'Object 1', another: create(Expando, { string: 'Object 2' }) }));
+
+    expect(readReference(obj, 'another')!.dxn.toString()).to.eq(`dxn:echo:@:${obj.another.id}`);
```

Comment:
is there a reason `obj.another` can't return the reference rather than needing this helper?

Reply:
The eventual goal is for it to do exactly that, this helper is in-place to allow us to experiment with the new API without breaking backwards compatibility.